src/MessagingAvatar/MessagingAvatar.tsx

```diff
@@ -4,23 +4,32 @@ import * as classNames from "classnames";
 import "./MessagingAvatar.less";
 
 const cssClasses = {
+  IMAGE: "Avatar--Image",
   CIRCLE: "Avatar--Circle",
   TEXT: "Avatar--Text",
 };
 
 type Props = {
   className?: string;
   text: string;
-  color?:
+  color:
```

Comment:
discussed with @adamvictorclever that this was meant to be required

Reply:
Can you verify that this isn't a breaking change on the Student or Teacher pages? Not that you shouldn't make it, just seems good to check.